src/cartesian/ErrorBar.tsx

```diff
@@ -52,23 +52,27 @@ interface ErrorBarProps extends InternalErrorBarProps {
 
 export type Props = SVGProps<SVGLineElement> & ErrorBarProps;
 
+ErrorBar.defaultProps = {
+  stroke: 'black',
+};
 export function ErrorBar(props: Props) {
   const {
-    offset,
-    layout,
-    width,
+    offset = 0,
```

Comment:
this causes visual regression https://www.chromatic.com/test?appId=63da8268a0da9970db6992aa&id=664cfc9eadd585053f5b885c

Since props are read from the React component before the component is actually called, these won't be set correctly